.gitignore

```diff
@@ -7,3 +7,4 @@ __pycache__/
 .coverage
 .coverage.*
 *.pyc
+.vscode/
```

Comment:
No issue with adding this but in general I usually create my VS Code workspace outside the project.

Reply:
ok